### PR TITLE
Create drug info script and url dictionary for patientslikeme.com

### PR DESCRIPTION
**Pull request by svillah**
- [x]  @M-Grab 

- [ ]  @SeanSiow 

drugInformation.py pulls drug information for a specific drug URL and exports to drugData.json 
drugsDict.py creates a dictionary that stores key: name of drug and value: URL for that drug

note: drugInformation.py may need to be updated when database is created

### CODE REVIEW

File: patientslikeme/drugInformation.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    dosages = soup.find_all("tr", {"data-yah-key":"dosage"})
+    ds = []
+    for dosage in dosages:
+        ds.append(dosage.get('data-yah-value'))
+    return ds[0]
+
+def overTheCounter(soup):
+    tag = soup.find("p", {"class":"margin-vertical-none"})
+    category = tag.find("a").string.encode('utf-8')
+    print category
```
Comment (M-Grab) [inline patientslikeme/drugInformation.py]: This is a debugging line, right? We probably don't want to be printing stuff to console live

File: patientslikeme/drugInformation.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    data["SideEffects"] = sideEffects(soup)
+    data["RecommendedDosage"] = recommendedDosage(soup) 
+    data["Source"] = url
+    # below are unavailable on patientslikeme.com
+    data["Interactions"] = None
+    data["ActiveIngredients"] = None
+    data["Warnings"] = None
+
+    # writes dictionary to json
+    json.dump(data, f, indent=4)
```
Comment (M-Grab) [inline patientslikeme/drugInformation.py]: magic number; use named constant?

File: patientslikeme/drugsDict.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+            name = brand.string.rstrip().encode('utf-8')
+            dictionary[name] = fullUrl
+
+#this will be our drug dicitonary
+drugDict = {}
+website = "https://www.patientslikeme.com/treatments/browse?cat="
+
+#creating dictionary for drugs
+drugDict = {}
+allCat = [1,2]
```
Comment (M-Grab) [inline patientslikeme/drugsDict.py]: magic numbers? Not sure what this variable is

File: patientslikeme/drugsDict.py [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+            # get URLs of all drugs
+            website = "https://www.patientslikeme.com/treatments/browse?cat=" + str(category).encode('utf-8') + "&page=" + str(pageNum).encode('utf-8')
+            soup = initializeSoup(website)
+            drugTable = soup.find("table", {"id":"tbl-treatments"})
+            drugs = drugTable.findChildren('tr')
+            disabled = soup.find("a", {"class":"button icon-only is-not-rwd next_page",
+                                   "disabled":"true"})    
+            #iterate through each drug on the page
+            for i in range(1, len(drugs)):
+                for drugUrl in drugs[i].findChildren("a"):
```
Comment (M-Grab) [inline patientslikeme/drugsDict.py]: Looks like the same code block as in line 47. Try to abstract the logic to another function?

File: patientslikeme/drugInformation.py [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+# get URLs of all drugs, note that this is only page one
+website = "https://www.patientslikeme.com/treatments/browse?cat=1"
```
Comment (M-Grab) [inline patientslikeme/drugInformation.py]: This line down seems like old code. Is it still relevant here or is stuff handled in `drugDic.py`?

**Review by M-Grab — CHANGES_REQUESTED** [review]
Good job, just some clean up required :)

File: patientslikeme/drugInformation.py (hunk 1 above)
Comment (svillah) [reply patientslikeme/drugInformation.py]: yes. reviewed & fixed!

File: patientslikeme/drugsDict.py (hunk 3 above)
Comment (SeanSiow) [reply patientslikeme/drugsDict.py]: okay I gave fixed the issue by creating constants of each category and assigning them with the appropriate numbers

https://gyazo.com/00587a63888c70aeb0339c04b77a89c3

File: patientslikeme/drugsDict.py (hunk 4 above)
Comment (SeanSiow) [reply patientslikeme/drugsDict.py]: abstract function created

**Review by M-Grab — APPROVED** [review]

**Pull request merged**